lib/service/services_cli.rb

```diff
@@ -180,40 +180,45 @@ def kill(targets, verbose: false)
     def take_root_ownership(service)
       return unless System.root?
 
-      chown "root", "admin", service.dest
-      plist_data = service.dest.read
-      plist = begin
-        Plist.parse_xml(plist_data)
-      rescue
-        nil
-      end
-      return unless plist
-
       root_paths = []
+      group = "root"
```

Comment:
I don't think we should set a default value here since both supported OSes have a different group.

Reply:
That is why I set `group = "admin"` in https://github.com/Homebrew/homebrew-services/blob/8a538aad9b58c0060aaa3d6b644156d319c17f07/lib/service/services_cli.rb#L187

Reply:
This would be better set in an `else` for readability IMO.

Reply:
Adjusted.